docs/markdown/Python/python-goals/python-test-goal.md

```diff
@@ -67,9 +67,28 @@ python_tests(
 )
 ```
 
-> 🚧 Avoid the `pytest-xdist` plugin
+> 🚧 How to use the `pytest-xdist` plugin
```

Comment:
Thoughts on this being a warning vs. the green "success" vs blue "info"?

Reply:
I feel good about blue - maybe we can make it green after it's been in the wild for awhile 😅

Reply:
You'll change the icon to this 📘